weblate/api/views.py

```diff
@@ -2213,33 +2213,46 @@ def translations(self, request: Request, *args, **kwargs):
         methods=["post"],
         request=CommentSerializer,
     )
-    @action(detail=True, methods=["post"], serializer_class=CommentSerializer)
+    @extend_schema(
+        description="Return a list of comments on the unit.", methods=["get"]
+    )
```

Comment:
I think the OpenAPI schema will be wrong, it needs to know it returns list of the comments.